native/cocos/bindings/jswrapper/napi/HelperMacros.h

```diff
@@ -0,0 +1,158 @@
+        se::internal::setReturnValue(state.rval(), retVal);                                           \
+        return retVal;                                                                                \
+    }
+
+#define SE_BIND_PROP_SET(funcName)                                                                    \
+    napi_value funcName##Registry(napi_env env, napi_callback_info info) {                            \
+        LOGI("SE_BIND_PROP_SET set %s", #funcName);                                                   \
+        napi_status status;                                                                           \
+        size_t      argc = 1;                                                                         \
+        napi_value  args[1];                                                                          \
+        napi_value  _this;                                                                            \
+        se::Value   data;                                                                             \
+        NODE_API_CALL(status, env, napi_get_cb_info(env, info, &argc, args, &_this, nullptr));        \
+        se::internal::jsToSeValue(args[0], &data);                                                    \
+        se::ValueArray args2;                                                                         \
+        args2.reserve(10);                                                                            \
+        args2.push_back(std::move(data));                                                             \
+        void *nativeThisObject;                                                                       \
+        napi_unwrap(env, _this, reinterpret_cast<void **>(&nativeThisObject));                        \
+        se::State state(nativeThisObject, args2);                                                     \
+        bool      ret = funcName(state);                                                              \
+        if (!ret) {                                                                                   \
+            SE_LOGE("[ERROR] Failed to invoke %s, location: %s:%d\n", #funcName, __FILE__, __LINE__); \
+        }                                                                                             \
+        return nullptr;                                                                               \
+    }
+
+#define SE_DECLARE_FUNC(funcName) \
+    napi_value funcName##Registry(napi_env env, napi_callback_info info)
+
+#define SE_BIND_FUNC(funcName)                                                                          \
+    napi_value funcName##Registry(                                                                      \
+        napi_env env, napi_callback_info info) {                                                        \
+        LOGI("%s", #funcName);                                                                          \
```

Comment:
Remove log

Reply:
At present, because NAPI is not perfect, there will be many problems. Every time you debug, you need to print the log to check the problem. Therefore, it can be temporarily placed, and it will be removed when it is stable in the later stage.

Reply:
Should not use LOGI, I means `INFO` level. Perhaps it should be LOGD (debug level).